app/templates/users/dashboard/base.html

```diff
@@ -10,7 +10,9 @@
     <img class="user-image" src="{{ url_for('main.identicon', text=identicode) }}" alt="User Image">
     <!-- hidden-xs hides the username on small devices so only the image appears. -->
     <span class="hidden-xs">
-      {% if not user.profile.first_name %}
+	  {% if not user.profile %}
+        {{ user.id.split('@')[0] }}
+      {% elif not user.profile.first_name %}
```

Comment:
ce n'est pas obligatoire d'avoir un first_name quand on crée un profil?

Reply:
J'ai rajouté ce test là parce qu'il y a une erreur récurrente au niveau de papertrail alert qui dit qu'un étudiant n'a pas de first name, c'est bizarre mais bon... Pour éviter ce soucis la j'ai passé ce test

Reply:
D'accord. Vaudrait mieux investiguer la vraie source du souci, mais tu peux au moins faire ça en attendant 🆗 ?
```suggestion
{% if user.profile and user.profile.first_name %}
    {{ "%s %s"|format(user.profile.first_name|capitalize, user.profile.name|capitalize)}}
{% else %}
    {{ user.id.split('@')[0] }}
{{ % endif % }}
```